### PR TITLE
Add simplify option in pytorch2onnx.py and Update doc

### PR DESCRIPTION
**Pull request by QingChuanWS**
This PR focus on add the simplify option in pytorch2onnx.py, which can simplify the model, and update relevant document in pytorch2onnx.md

### CODE REVIEW

File: docs/tutorials/pytorch2onnx.md [hunk 1]
```diff
@@ -57,21 +59,22 @@ python tools/pytorch2onnx.py \
...
 
-|    Model     |                            Config                            | Note |
-| :----------: | :----------------------------------------------------------: | :--: |
-| MobileNetV2  |    `configs/mobilenet_v2/mobilenet_v2_b32x8_imagenet.py`     |      |
-|    ResNet    |          `configs/resnet/resnet18_b16x8_cifar10.py`          |      |
-|   ResNeXt    |     `configs/resnext/resnext50_32x4d_b32x8_imagenet.py`      |      |
-|  SE-ResNet   |       `configs/seresnet/seresnet50_b32x8_imagenet.py`        |      |
-| ShuffleNetV1 | `configs/shufflenet_v1/shufflenet_v1_1x_b64x16_linearlr_bn_nowd_imagenet.py` |      |
-| ShuffleNetV2 | `configs/shufflenet_v2/shufflenet_v2_1x_b64x16_linearlr_bn_nowd_imagenet.py` |      |
+|    Model     |                            Config                            | Dynamic Shape | Note |
```
Comment (RunningLeon) [inline docs/tutorials/pytorch2onnx.md:70]: Batch Inference could be also added here.

File: docs/tutorials/pytorch2onnx.md [hunk 2]
```diff
@@ -57,21 +59,22 @@ python tools/pytorch2onnx.py \
     --output-file checkpoints/resnet/resnet18_b16x8_cifar10.onnx \
     --dynamic-shape \
     --show \
+    --simplify \
```
Comment (RunningLeon) [inline docs/tutorials/pytorch2onnx.md:62]: Please verify whether `--dynamic-shape` and `--simplify` could work together.

Comment (QingChuanWS) [reply docs/tutorials/pytorch2onnx.md:62]: `--dynamic-shape` and `--simplify` could work together.

File: docs/tutorials/pytorch2onnx.md [hunk 3]
```diff
@@ -57,21 +59,22 @@ python tools/pytorch2onnx.py \
...
 
-|    Model     |                            Config                            | Note |
-| :----------: | :----------------------------------------------------------: | :--: |
-| MobileNetV2  |    `configs/mobilenet_v2/mobilenet_v2_b32x8_imagenet.py`     |      |
-|    ResNet    |          `configs/resnet/resnet18_b16x8_cifar10.py`          |      |
-|   ResNeXt    |     `configs/resnext/resnext50_32x4d_b32x8_imagenet.py`      |      |
-|  SE-ResNet   |       `configs/seresnet/seresnet50_b32x8_imagenet.py`        |      |
-| ShuffleNetV1 | `configs/shufflenet_v1/shufflenet_v1_1x_b64x16_linearlr_bn_nowd_imagenet.py` |      |
-| ShuffleNetV2 | `configs/shufflenet_v2/shufflenet_v2_1x_b64x16_linearlr_bn_nowd_imagenet.py` |      |
+|    Model     |                            Config                            | Batch Infer | Dynamic Shape | Note |
```
Comment (RunningLeon) [inline docs/tutorials/pytorch2onnx.md:70]: Batch Inference

File: docs/tutorials/pytorch2onnx.md [hunk 4]
```diff
@@ -57,21 +59,22 @@ python tools/pytorch2onnx.py \
...
-| :----------: | :----------------------------------------------------------: | :--: |
-| MobileNetV2  |    `configs/mobilenet_v2/mobilenet_v2_b32x8_imagenet.py`     |      |
-|    ResNet    |          `configs/resnet/resnet18_b16x8_cifar10.py`          |      |
-|   ResNeXt    |     `configs/resnext/resnext50_32x4d_b32x8_imagenet.py`      |      |
-|  SE-ResNet   |       `configs/seresnet/seresnet50_b32x8_imagenet.py`        |      |
-| ShuffleNetV1 | `configs/shufflenet_v1/shufflenet_v1_1x_b64x16_linearlr_bn_nowd_imagenet.py` |      |
-| ShuffleNetV2 | `configs/shufflenet_v2/shufflenet_v2_1x_b64x16_linearlr_bn_nowd_imagenet.py` |      |
+|    Model     |                            Config                            | Batch Inference | Dynamic Shape | Note |
+| :----------: | :----------------------------------------------------------: | :-------------: | :-----------: | ---- |
+| MobileNetV2  |    `configs/mobilenet_v2/mobilenet_v2_b32x8_imagenet.py`     |        N        |       Y       |      |
```
Comment (RunningLeon) [inline docs/tutorials/pytorch2onnx.md:72]: Batch inference should be OK for the exported ONNX models with ONNXRuntime backend. You could verify them.

**Review by RunningLeon — APPROVED** [review]
LGTM

**Pull request merged**